components/feedstock/stores.js

```diff
@@ -87,39 +101,49 @@ const Store = ({ dataset }) => {
           <Code language='python'>{getSnippet(url)}</Code>
 
           <Button
-            as={Link}
-            href={`https://data-viewer-git-katamartin-pyramid-maps-carbonplan.vercel.app/?dataset=${pyramid || url}`}
+            href={`https://ncview-js.staging.carbonplan.org/?dataset=${
+              pyramid || url
+            }`}
             target='_blank'
             rel='noopener noreferrer'
-            suffix={<RotatingArrow sx={{ ml: '2px' }} />}
+            suffix={
+              <RotatingArrow
+                sx={{
+                  transform: ['translateY(-6%)'],
+                }}
+              />
+            }
             sx={{
+              color: color,
+              fontSize: [0, 0, 0, 1],
+              letterSpacing: 'mono',
+              textTransform: 'uppercase',
+              fontFamily: 'mono',
               mt: 3,
               mb: 2,
-              fontSize: 0,
-              fontFamily: 'mono',
-              color: 'secondary',
-              textTransform: 'uppercase',
             }}
           >
             Open in Data Viewer
           </Button>
         </Box>
       </AnimateHeight>
-    </Box>
+    </Flex>
   )
 }
 
-export const Stores = ({ stores }) => {
+export const Stores = ({ stores, color }) => {
   return (
-    <Box>
-      <Text sx={{ fontSize: 1, fontWeight: 'bold', color: 'muted' }}>
-        {stores.length > 1 ? 'Stores:' : 'Store:'}
-      </Text>
-      <Flex sx={{ flexDirection: 'column', ml: 4 }}>
-        {stores.map((store) => (
-          <Store key={store.id} dataset={store} />
-        ))}
-      </Flex>
-    </Box>
+    <Flex
+      sx={{
+        flexDirection: 'column',
+        gap: 4,
```

Comment:
Another tweak along the same lines:
```suggestion
        gap: [3, 3, 3, 4],
```